app/helpers/handlers.js

```diff
@@ -200,17 +200,7 @@ const getPage = async (question, request, h) => {
       }
     }
     // case 'score':
-    case 'living-space-3m2':
```

Comment:
You're right that we don't need this switch case as we only have one yar key, but we should still reset the livingSpace yarKey if a user changes between the 3 pages